src/query.js

```diff
@@ -2920,6 +2928,9 @@ function wrap(obj) {
   } else if (obj instanceof Uint8Array || obj instanceof ArrayBuffer) {
     return new values.Bytes(obj)
   } else if (typeof obj === 'object') {
+    if (obj.constructor.name === 'Expr') {
+      console.log(obj.constructor.name, Object.keys(obj), obj)
```

Comment:
Could we remove this?

Reply:
Dang, good catch. I'll remove it in a minute.